pkg/standalone/stop_windows.go

```diff
@@ -29,13 +28,19 @@ func Stop(appID string) error {
 
 	for _, a := range apps {
 		if a.AppID == appID {
-			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.PID))
-			eventHandle, err := windows.OpenEvent(windows.EVENT_MODIFY_STATE, false, &eventName[0])
-			if err != nil {
-				return err
+			var pid string
+			// Kill the Daprd process if Daprd was started without CLI, otherwise
+			// kill the CLI process which also kills the associated Daprd process.
+			if a.CliPID == 0 {
+				pid = fmt.Sprintf("%v", a.DaprdPID)
+			} else {
+				pid = fmt.Sprintf("%v", a.CliPID)
 			}
 
-			err = windows.SetEvent(eventHandle)
+			// /F forcefully end the process
+			// /T end child processes along with the main process
+			_, err := utils.RunCmdAndWait("taskkill", "/F", "/T", "/PID", pid)
```

Comment:
do we need `/F`? How does it translate in terms of Signals? is it SIGKILL or SIGTERM? We need it to send a SIGTERM or SIGINT event and not SIGKILL to gracefully shutdown the process ...